tests/ma_bots2/country2_bots/test_c_2_1_lab.py

```
@@ -0,0 +1,239 @@
+    "dominican republic": "جمهورية الدومينيكان",
+    "dutch east indies": "الهند الشرقية الهولندية",
+    "dutch republic": "جمهورية هولندا",
+    "east midlands": "ميدلاند الشرقية",
+    "european union": "الاتحاد الأوروبي",
+    "falkland islands": "جزر فوكلاند",
+    "faroe islands": "جزر فارو",
+    "fatimid caliphate": "الدولة الفاطمية",
+    "federated states-of micronesia": "ولايات ميكرونيسيا المتحدة",
+    "gaza strip": "قطاع غزة",
+    "holy roman empire": "الإمبراطورية الرومانية المقدسة",
+    "israel–hamas war": "الحرب الفلسطينية الإسرائيلية",
```

Ruff (0.14.11)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)